packages/dnb-eufemia/src/components/avatar/Avatar.tsx

```diff
@@ -82,6 +82,18 @@ export interface AvatarProps
    * Default: null
    */
   hasLabel?: boolean
+
+  /**
+   * Define a custom background color, instead of a variant. Use a Eufemia color.
+   * Default: null
```

Comment:
Seems like the default to ´undefined´ instead of `null`? 🤔

```suggestion
   * Default: undefined
```